### PR TITLE
fix wallet migration test based on new changes

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**
with changes in view port in code structure, this test needs to be updated
Screenshot 2024-07-26 at 12 07 03 PM

### CODE REVIEW

**Review by rajendra-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/smoke/Wallets/walletMigrationSuccess.cy.js [hunk 1]
```diff
@@ -126,25 +126,23 @@ countyCode.forEach((countyCode) => {
...
-    const size = ['small', 'desktop']
-    size.forEach((size) => {
-      it('should be able to migrate Fiat account', () => {
-        cy.c_login()
-        const isMobile = size == 'small' ? true : false
-        cy.c_visitResponsive('/', size)
-        migratetoWallet(size)
-        checkFiatWalletExistance(size)
-        checkDemoExistance(size)
+    const viewSize = ['mobile', 'desktop']
```
Comment (bilalmajid-deriv) [inline cypress/e2e/smoke/Wallets/walletMigrationSuccess.cy.js:129]: can you please help to replace viewSize with size to mantain uniformity through out the project and follow pattern mentioned here
https://deriv-group.slack.com/archives/C06GQ762W4C/p1721971519937969

**Review by meeyun-deriv — APPROVED** [review]
LGTM apart from Bilal comment

File: cypress/e2e/smoke/Wallets/walletMigrationSuccess.cy.js (hunk 1 above)
Comment (afsaneh-rezaeinejad-deriv) [reply cypress/e2e/smoke/Wallets/walletMigrationSuccess.cy.js:129]: @bilalmajid-deriv code is updated

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Pull request merged**